monkestation/code/modules/power/rbmk/rbmk.dm

```diff
@@ -0,0 +1,1091 @@
+	CO2: Super effective shutdown gas for runaway reactions. MASSIVE RADIATION PENALTY!
+	Pluoxium: Same as N2, but no cancer-rads!
+
+	Permeability Type (Coolant loop speed):
+	BZ: Increases your reactor's ability to transfer its heat to the coolant, thus letting you cool it down faster (but your output will get hotter)
+	Water Vapour: More efficient permeability modifier
+	Hyper Noblium: Extremely efficient permeability increase. (10x as efficient as bz)
+
+	Depletion type:
+	Nitryl: When you need weapons grade plutonium yesterday. Causes your fuel to deplete much, much faster. Not a huge amount of use outside of sabotage.
+
+Sabotage:
+	Meltdown:
+	Flood reactor moderator with plasma, they won't be able to mitigate the reaction with control rods.
+	Shut off coolant entirely. Raise control rods.
+	Swap all fuel out with spent fuel, as it's way stronger.
+
+	Blowout:
+	Shut off exit valve for quick overpressure.
+	Cause a pipefire in the coolant line (LETHAL).
+	Tack heater onto coolant line (can also cause straight meltdown)
+
+Tips:
+Be careful to not exhaust your plasma supply. I recommend you DON'T max out the moderator input when youre running plasma + o2, or you're at a tangible risk of running out of those gasses from atmos.
+The reactor CHEWS through moderator. It does not do this slowly. Be very careful with that!
+
+//Remember kids. If the reactor itself is not physically powered by an APC, it cannot shove coolant in!
+
+*/
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor
+	name = "Advanced Gas-Cooled Nuclear Reactor"
+	desc = "A tried and tested design which can output stable power at an acceptably low risk. The moderator can be changed to provide different effects."
+	icon = 'monkestation/icons/obj/machinery/rbmk.dmi'
+	icon_state = "reactor_map"
+	pixel_x = -32
+	pixel_y = -32
+	processing_flags = START_PROCESSING_MANUALLY
+	density = FALSE //It burns you if you're stupid enough to walk over it.
+	anchored = TRUE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+	light_color = LIGHT_COLOR_CYAN
+	dir = 8 //Less headache inducing :))
+	var/id = null //Change me mappers
+
+	//Variables essential to operation
+	var/temperature = 0 //Lose control of this -> Meltdown
+	var/vessel_integrity = 400 //How long can the reactor withstand overpressure / meltdown? This gives you a fair chance to react to even a massive pipe fire
+	var/pressure = 0 //Lose control of this -> Blowout
+	var/K = 0 //Rate of reaction.
+	var/desired_k = 0
+	var/control_rod_effectiveness = 0.65 //Starts off with a lot of control over K. If you flood this thing with plasma, you lose your ability to control K as easily.
+	var/power = 0 //0-100%. A function of the maximum heat you can achieve within operating temperature
+	var/power_modifier = 1 //Upgrade me with parts, science! Flat out increase to physical power output when loaded with plasma.
```

Comment:
If internal parts can be upgraded, the description should probably reflect this

Reply:
Ah that was labeled wrong, that variable is used with moderator gasses for determining power output. NSV13 must of been planning to, but never got around to adding it.